sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_differentiate_output_models_trained_with_and_without_labels_async.py

```diff
@@ -75,8 +81,19 @@ async def recognize_custom_forms(self):
                         field.confidence
                     ))
 
-            print("------------------------------------------------------------------")
-            print("-------Recognizing forms with models trained without labels-------")
+            # Find a specific labeled field. Substitute "Merchant" with your specific training-time label
```

Comment:
nit: you might want to pull out "Merchant" into a variable, i.e.

`training_time_label = 'Merchant'`, and use training_time_label throughout the test instead of "Merchant"

Reply:
updated